extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -452,17 +453,17 @@ private function processInTrackPicture() {
 	private function processInTrackCategory() {
 		// BADGE_CATEGORY
 		$insertedCategories = Wikia::getVar( 'categoryInserts' );
-		if ( !empty( $insertedCategories ) && is_array( $insertedCategories ) ) {
+		if( !empty( $insertedCategories ) && is_array( $insertedCategories ) ) {
 			$this->incrementCounterBy( BADGE_CATEGORY, count( $insertedCategories ) );
 		}
 	}
 
 	private function processInTrackBlogPost() {
 		//BADGE_BLOGPOST
 		//is defined check if required because blogs are not enabled everywhere
-		if ( defined( 'NS_BLOG_ARTICLE' ) && $this->mTitle->getNamespace() == NS_BLOG_ARTICLE ) {
-			if ( $this->mTitle->getBaseText() == $this->mUser->getName() ) {
-				if ( $this->mStatus->value['new'] == true ) {
+		if( defined( 'NS_BLOG_ARTICLE' ) && $this->mTitle->getNamespace() == NS_BLOG_ARTICLE ) {
+			if( $this->mTitle->getBaseText() == $this->mUser->getName() ) {
+				if( $this->mStatus->value[ 'new' ] == true ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1066)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "true" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1125)